MoreLinq.Test/ReturnTest.cs

```diff
@@ -0,0 +1,145 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2019 Mitch Bodmer. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+using System;
+
+namespace MoreLinq.Test
+{
+    using NUnit.Framework;
+    using System.Collections.Generic;
+
+    class ReturnTest
+    {
+        [Test]
+        public void TestResultingSequenceContainsSingle()
+        {
+            Assert.That(MoreEnumerable.Return(new object()).Count(), Is.EqualTo(1));
+        }
+
+        [Test]
+        public void TestResultingSequenceContainsTheItemProvided()
+        {
+            var item = new object();
+            Assert.That(MoreEnumerable.Return(item), Has.Member(item));
+        }
+
+        [Test]
+        public void TestResultingListHasCountOne()
+        {
+            Assert.That(((IList<object>) MoreEnumerable.Return(new object())).Count, Is.EqualTo(1));
+        }
+
+        [Test]
+        public void TestContainsReturnsTrueWhenTheResultingSequenceContainsTheItemProvided()
+        {
+            var item = new object();
+            Assert.That(MoreEnumerable.Return(item).Contains(item), Is.True);
+        }
+
+        [Test]
+        public void TestContainsDoesNotThrowWhenTheItemContainedIsNull()
+        {
+            Assert.That(() => MoreEnumerable.Return(new object()).Contains(null), Throws.Nothing);
+        }
+
+        [Test]
+        public void TestContainsDoesNotThrowWhenTheItemProvidedIsNull()
+        {
+            Assert.That(() => MoreEnumerable.Return<object>(null).Contains(new object()), Throws.Nothing);
+        }
+
+        [Test]
+        public void TestIndexOfDoesNotThrowWhenTheItemProvidedIsNull()
+        {
+            Assert.That(() => ((IList<object>) MoreEnumerable.Return<object>(null)).IndexOf(new object()),
+                Throws.Nothing);
+        }
+
+        [Test]
+        public void TestIndexOfDoesNotThrowWhenTheItemContainedIsNull()
+        {
+            Assert.That(() => ((IList<object>)MoreEnumerable.Return<object>(new object())).IndexOf(null),
+                Throws.Nothing);
+        }
+
+        [Test]
+        public void TestCopyToSetsTheValueAtTheIndexToTheItemContained()
```

Comment:
This test can be a little more thorough. I would initialize `array` with 3 objects and ask the collection to copy to offset 1 and then test that the middle item was changed to the one in the collection while the first and last items in the array remained the same.